addons/web_editor/static/src/components/media_dialog/media_dialog.scss

```diff
@@ -1,114 +1,55 @@
-    }
-
-    .o_we_transition_ease {
-        transition: all ease 0.25s;
-    }
 
     .o_we_existing_attachments {
-        min-height: $min-row-height + px;
-
-        .o_we_attachment_placeholder {
-            flex-grow: $min-row-height;
-            flex-basis: $min-row-height + px;
-        }
-
         .o_existing_attachment_cell {
-            cursor: pointer;
-            margin: 1px;
-
             &.o_we_image {
                 position: fixed !important;
                 opacity: 0;
-                transition: opacity 0.5s;
-                transition-delay: 0.5s;
+                transition: opacity 0.5s ease 0.5s;
 
                 &.o_loaded {
                     position: relative !important;
                     opacity: 1;
+
+                    img{
+                        height: 120px;
+                    }
                 }
 
                 .o_we_media_dialog_img_wrapper {
                     @extend %o-preview-alpha-background;
-
-                    img {
-                        width: 100%;
-                    }
                 }
             }
 
             .o_existing_attachment_remove {
-                background-color: rgba(white, 0.4);
-                cursor: pointer;
-                opacity: 0;
                 transition: color 0.2s ease;
-            }
-
-            .o_existing_attachment_remove {
-                @include o-position-absolute($top: 0, $right: 0);
                 z-index: 1;
-                border-radius: 0 0 0 2px;
+                border-radius: 0 0 0 $border-radius;
                 &:hover {
-                    color: $o-we-color-danger;
+                    color: $o-danger;
                 }
             }
 
-            .o_file_name {
-                @include o-text-overflow;
-            }
-
             &:hover {
                 .o_existing_attachment_remove {
-                    opacity: 1;
-                }
-                &.o_we_attachment_highlight, .o_we_attachment_highlight {
-                    border-color: $card-border-color;
-                    box-shadow: 0px 0px 2px 2px $card-border-color;
+                    opacity: 1 !important;
                 }
             }
-
-            &.o_we_attachment_optimized .badge {
-                position: absolute;
-                bottom: 0;
-                right: 0;
-                margin: 2px;
-            }
-
-            .o_we_media_author {
-                font-size: 11px;
-                @include o-position-absolute($bottom: 0, $left: 0, $right: 0);
-                white-space: nowrap;
-                overflow: hidden;
-                text-overflow: ellipsis;
-                text-align: center;
-                background-color: rgba(255, 255, 255, .7);
-            }
         }
     }
 
     .o_we_attachment_selected {
         @include o-we-active-wrapper($top: 5px, $left: 5px);
+        .o_we_media_dialog_img_wrapper{
+            img{
+                height: calc(120px - 6px) !important;
```

Comment:
Sounds everything quite arbitrary ;)

Let's create a local variables for the img's height, I sow that 120px is already used somewhere else indeed.
For local variables we just use the dash prefix, in your case something like `$-imgHeight` should be ok.

also, what is `6px`? Why you need to use `calc()`?